core/main/src/firebolt/handlers/localization_rpc.rs

```diff
@@ -48,7 +52,7 @@ use serde::Deserialize;
 #[derive(Deserialize, Debug)]
 pub struct SetMapEntryProperty {
     pub key: String,
-    pub value: String,
+    pub value: Value,
```

Comment:
This fixes firebolt 1.5.0 support for string/int/bool values. Ticket includes request/responses to confirm fix and compatibility with any pre-existing stored additional info.